lib/mix/lib/mix/compilers/elixir.ex

```diff
@@ -188,8 +189,9 @@ defmodule Mix.Compilers.Elixir do
     Mix.shell.info "Compiled #{source}"
   end
 
-  defp each_long_compilation(source) do
-    Mix.shell.info "Compiling #{source} (it's taking more than 5s)"
+  defp each_long_compilation(source, threshold_in_ms) do
+    threshold_in_s = round(div(threshold_in_ms, 1_000))
```

Comment:
Unless we want to show "taking more than 5.112s", which I really dislike :) But if we're ok with that then no we don't need `round/1`.